crates/ra_parser/src/grammar.rs

```diff
@@ -93,12 +93,12 @@ pub(crate) mod fragments {
                 // https://doc.rust-lang.org/reference/paths.html#simple-paths
                 // The start of an meta must be a simple path
                 match p.current() {
-                    IDENT | T![::] | T![super] | T![self] | T![crate] => p.bump(),
+                    IDENT | T![::] | T![super] | T![self] | T![crate] => p.bump_any(),
                     T![=] => {
-                        p.bump();
+                        p.bump_any();
```

Comment:
This and all the `if p.at` could use `bump` right?

Reply:
Yeah, ideally we should remove `bump_any` completely, but I want to do that after #1801